extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -55,10 +55,24 @@
   top: 0 !important;
   height: 50px !important;
   background: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0));
-  transition: opacity 1s;
   z-index: 2;
 }
 
+.i-amphtml-lbv-controls {
+  opacity: 0;
+  -webkit-animation: fadeIn ease-in 0.4s 1 forwards;
```

Comment:
no need for prefixing, our CSS compiler will do that

Reply:
Cool! removed.